include/rdm/responder.h

```diff
@@ -80,6 +90,92 @@ etcpal_error_t rdmresp_unpack_command(const RdmBuffer *buffer, RdmCommand *cmd);
 bool rdmresp_is_non_disc_command(const RdmBuffer *buffer);
 etcpal_error_t rdmresp_pack_response(const RdmResponse *resp_data, RdmBuffer *buffer);
 
+typedef enum
+{
+  kRespAck = E120_RESPONSE_TYPE_ACK,
+  kRespAckTimer = E120_RESPONSE_TYPE_ACK_TIMER,
+  kRespNackReason = E120_RESPONSE_TYPE_NACK_REASON,
+  kRespAckOverflow = E120_RESPONSE_TYPE_ACK_OVERFLOW,
+  kRespNoSend
+} resp_process_result_t;
+
+typedef enum
+{
+  kRespTypeRDM,
+  kRespTypeController,
+  kRespTypeBroker,
+  kRespTypeDevice
+} resp_type_t;
+
+typedef struct PidHandlerData
+{
+  uint8_t port;
+  uint8_t cmd_class;
+  uint8_t sub_device;
+  size_t overflow_index;
+  void *context;
+
+  const RdmParamData *pd_in;
+  RdmParamData *pd_out;
+} PidHandlerData;
+
+typedef struct RdmPidHandlerEntry
+{
+  uint16_t pid;
+  resp_process_result_t (*handler)(PidHandlerData *data);
+  uint32_t flags;
+} RdmPidHandlerEntry;
+
+typedef struct GetNextQueueMessageData
+{
+  uint8_t port;
+  uint8_t status_type;
+  uint8_t *response_type;
+  uint16_t *sub_device;
+  uint8_t *cmd_class;
+  uint16_t *param_id;
+  RdmParamData *pd;
+} GetNextQueueMessageData;
```

Comment:
I think it should be `GetNextQueuedMessageData` (Queued in the past tense)

Reply:
Done.